<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdDialog</class>
 <widget class="QDialog" name="CmdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QComboBox" name="comboBox_type">
     <item>
      <property name="text">
       <string>Одноэлементная команда (45, 58)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Двухэлементная команда (46, 59)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Команда пошагового регулирования (47, 60)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Команда уставки, нормализованное значение (48, 61)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Команда уставки, масштабированное значение (49, 62)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Команда уставки, короткое число с плавающей точкой (50, 63)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Строка из 32 битов (51, 64)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_timestamp">
     <property name="text">
      <string>С меткой времени</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>IOA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_ioa">
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Значение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_value"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Активировать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
